Classes/PhpBench/FrameworkEnabledBenchmark.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * (c) Contributors of the Neos Project - www.neos.io
+ * Please see the LICENSE file which was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\BuildEssentials\PhpBench;
+
+use Neos\BuildEssentials\TestableFramework;
+use Neos\Flow\Core\Bootstrap;
+use Neos\Flow\Testing\RequestHandler\RuntimeSequenceInvokingRequestHandler;
+
+/**
+ * Tools for https://github.com/phpbench/phpbench
+ */
+class FrameworkEnabledBenchmark
+{
+    protected Bootstrap $flowBootstrap;
```

Comment:
instead of having to write

```php
$this->flowBootstrap->getObjectManager()->get(ResourceRepository::class);
```

i would favour a helper like we do for the behat tests:

```php
/**
 * Helper for fetching dependencies
 *
 * @template T of object
 * @param class-string<T> $className
 *
 * @return T
 *
 * @api
 */
final protected function getObject(string $className): object
{
    if (!isset($this->flowBootstrap)) {
        throw new \RuntimeException('Could not get object, because flow is not initialized.', 1698765963);
    }
    return $this->flowBootstrap->getObjectManager()->get($className);
}
```

Reply:
I don't know, especially for testing I find it much more explicit to do this way. Especially for the benchmarks, I do not want any "hidden" code happening.